frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -184,6 +186,30 @@ export class ClusterOverview extends React.Component {
       });
   }
 
+  fetchAlerts() {
+    const url = `${geAlertManagerBaseURL()}/api/v2/alerts`;
+    coFetchJSON(url)
+      .then(alertsResponse => {
+        if (this._isMounted) {
+          this.setState({
+            alertsResponse,
+          });
+        }
+      })
+      .catch(error => {
+        if (this._isMounted) {
+          this.setState({
+            alertsResponse: error,
```

Comment:
This will throw a object But we expecting array in alert component in web-ui-component.

Reply:
good catch. Fixed in web-ui-components PR